### DISCUSSION TITLE
How to switch between multiple API gateways in runtime?

### DISCUSSION THREAD

**Discussion by mivinci** [sveltejs/kit#7282] [Q&A]
Hey guys. I have an API gateway for development and several for production.

- mock.example.com
- a.example.com
- b.example.com
- c.example.com

and I want to switch between them in runtime by like using a `select` tag. So what is the best way to implement this with SvelteKit?

**Comment by mustofa-id** [comment]
You may switch them on [handleFetch](https://kit.svelte.dev/docs/hooks#server-hooks-handlefetch)
